clang/lib/Sema/SemaChecking.cpp

```diff
@@ -1981,6 +1981,29 @@ static bool OpenCLBuiltinToAddr(Sema &S, unsigned BuiltinID, CallExpr *Call) {
   return false;
 }
 
+// In OpenCL, __builtin_alloca_* should return a pointer to address space
+// that corresponds to the stack address space i.e private address space.
+static bool OpenCLBuiltinAllocaAddrSpace(Sema &S, CallExpr *TheCall) {
```

Comment:
Just a nit, since this function also has non-OpenCL specific code, it would make more sense to remove OpenCL from the name.

Reply:
This rename wasn't done. Also should start with lowercase letter